networks/known_networks_test.go

```diff
@@ -0,0 +1,85 @@
+package networks
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/logging"
+)
+
+func TestMain(m *testing.M) {
+	logging.Init()
+	os.Exit(m.Run())
+}
+
+func TestMustGetSelectedNetworksFromEnv_Missing_SELECTED_NETWORKS(t *testing.T) {
+	defer os.Clearenv()
+
+	require.Panics(t, func() {
+		MustGetSelectedNetworksFromEnv()
+	})
+}
+
+func TestMustGetSelectedNetworksFromEnv_Missing_HTTP_URLS(t *testing.T) {
+	defer os.Clearenv()
+
+	networkKey := "ARBITRUM_GOERLI"
+
+	os.Setenv("SELECTED_NETWORKS", networkKey)
```

Comment:
https://pkg.go.dev/testing#T.Setenv
Using t.Setenv will set the environment variable within the scope of the test and the test will clear it automatically without the need to defer the cleanup.

Reply:
great idea! Pushed a new commit